streams/ma.m3u

```diff
@@ -27,7 +27,7 @@ http://streaming1.medi1tv.com/live/smil:medi1tv.smil/playlist.m3u8
 https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/tamazight_tv8_snrt/hls_snrt/tamazight_tv8_snrt.m3u8
 #EXTINF:-1 tvg-id="TeleMaroc.ma" status="online",Télé Maroc (1080p)
 https://api.new.livestream.com/accounts/27130247/events/8196478/live.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",Al Aoula International (360p)
+#EXTINF:-1 tvg-id="AlAoula.ma" status="blocked",Al Aoula International (360p)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="AlAoulaEurope.ma" status="blocked",Al Aoula International (360p)
```
Consider using the European prefix for the International stream as they have their local channel too (the same works for 2M). It is the only channel from the SNRT to have this subtility as it's the first channel.